### ISSUE TITLE
Documentation

### ISSUE THREAD

**Issue by MaximMoinat** [eseraygun/python-alignment#12]
Does anyone know whether there is any documentation available on the full functionalities? Maybe someone has something local that he/she is willing to share? So far, I only found the example code snippets that help you to get started.

Especially interested in a short explanation on the differences between:
- `GlobalSequenceAlignment`
- `StrictGlobalSequenceAlignment`
- `LocalSequenceAlignment`

**Comment by reynoldsnlp** [comment]
I agree that this would be nice. Even just some simple docstrings for each class.
